ui/raidboss/data/06-ew/trial/hydaelyn-ex.txt

```diff
@@ -0,0 +1,204 @@
+# Hydaelyn Extreme
```

Comment:
Did you use any `-ii` parameters here when making this timeline? If so, could you list them here in a comment, e.g. `# -ii 1234 5678`?

Reply:
I did not use any -ii parameters as I am not aware of how those are set.

Reply:
Did you use make_timeline, at least? Or did you do this all manually? <_<

Reply:
I start with make_timeline, then check for which spells are redundant. Parts where time needs to be incremented for patterns/loops or windows I have done in a spreadsheet or regex find/replace.

Still learning more about how to handle patterns and pathfinding. Trial 1 did not have any patterns other than during adds that I noticed. For trial two I knew there were some patterns and had to dig out some other logs from fflogs to make sure of the patterns and find that they use different spell ids for some patterns. I compared make_timeline outputs to find the patterns. I merged make_timeline output for blocks of the pattern as I did not have logs from every possible outcome.